apps/frontend/src/app/5_pages/AaveReserveOverviewPage/components/TimeRangeButtons/TimeRangeButtons.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { FC, useState } from 'react';
+
+import {
+  ESupportedTimeRanges,
+  ReserveRateTimeRange,
+} from '../../../../../hooks/aave/useAaveReservesHistory';
+
+interface TimeRangeButtonsProps {
+  onChange: (range: ReserveRateTimeRange) => void;
+}
+
+export const TimeRangeButtons: FC<TimeRangeButtonsProps> = ({ onChange }) => {
+  const [activeRange, setActiveRange] = useState<ReserveRateTimeRange>(
+    ESupportedTimeRanges.OneMonth,
```

Comment:
I think this should be `ReserveRateTimeRange.OneMonth` with an enum

Reply:
`ReserveRateTimeRange` is the type to express 3 posibilities from the enum ESupportedTimeRanges